src/documentation/examples/Layout.tsx

```diff
@@ -0,0 +1,18 @@
+import Vue from "vue";
+import Component from "vue-class-component";
+
+import Layout from "@/components/Layout";
+import LayoutItem from "@/components/Layout/LayoutItem";
+
+@Component
+export default class App extends Vue {
```

Comment:
Please add more examples to see what a user can really do with these components.